service_container/factories.rst

```diff
@@ -100,6 +100,80 @@ create its object:
     the configured class name may be used by compiler passes and therefore
     should be set to a sensible value.
 
+Using the Class as Factory Itself
+---------------------------------
+
+When the static factory method is on the same class as the created instance,
+the class name can be omitted from the factory declaration.
+Let's suppose the ``NewsletterManager`` class has a ``create()`` method that needs
+to be called to create the object and need a sender::
```

Comment:
needs

Reply:
really?

Reply:
yes - the subject is the method, no?

Reply:
You are right, but needs twice sounds a bit weird maybe

Reply:
accepts, expects, etc